addons/test_mail_full/tests/test_sms_composer.py

```diff
@@ -656,3 +656,20 @@ def test_composer_template_context_action(self):
             test_record_2.customer_id, None,
             content="Hello %s ceci est en français." % test_record_2.display_name
         )
+
+    def test_composer_sending_with_no_number_field(self):
```

Comment:
It should be in 'TestSMSComposerComment' class, this one is the class about mass mode tests. Keeping tests organized is important to better know what are the tested and untested use cases :)

Would rename to 'test_composer_comment_nofield' and put it after 'test_composer_comment_field_w_template' test.